test/support/test_config.h

```diff
@@ -184,10 +184,10 @@
 #define _PSTL_ICC_TEST_COMPLEX_POW_SCALAR_COMPLEX_PASS_BROKEN_TEST_EDGES  _PSTL_TEST_COMPLEX_OP_BROKEN_IN_INTEL_LLVM_COMPILER
 #define _PSTL_ICC_TEST_COMPLEX_NORM_MINUS_INF_NAN_BROKEN_TEST_EDGES       _PSTL_TEST_COMPLEX_OP_BROKEN_IN_INTEL_LLVM_COMPILER
 #define _PSTL_ICC_TEST_COMPLEX_POLAR_BROKEN_TEST_EDGES                    _PSTL_TEST_COMPLEX_OP_BROKEN_IN_INTEL_LLVM_COMPILER
-#define _PSTL_ICC_TEST_COMPLEX_EXP_BROKEN_TEST_EDGES                     (20240201 < __INTEL_LLVM_COMPILER && __INTEL_LLVM_COMPILER <= _PSTL_TEST_LATEST_INTEL_LLVM_COMPILER)
-#define _PSTL_TEST_COMPLEX_ACOS_BROKEN_IN_KERNEL                         (__INTEL_LLVM_COMPILER <= 20240200 && __SYCL_DEVICE_ONLY__)
-#define _PSTL_TEST_COMPLEX_EXP_BROKEN                                    (__INTEL_LLVM_COMPILER <= 20240200 && __SYCL_DEVICE_ONLY__)
-#define _PSTL_TEST_COMPLEX_TANH_BROKEN_IN_KERNEL                         (__INTEL_LLVM_COMPILER <= 20240200 && __SYCL_DEVICE_ONLY__)
+#define _PSTL_ICC_TEST_COMPLEX_EXP_BROKEN_TEST_EDGES                     (20240201 < __INTEL_LLVM_COMPILER && __INTEL_LLVM_COMPILER < 20250100)
+#define _PSTL_TEST_COMPLEX_ACOS_BROKEN_IN_KERNEL                         (__INTEL_LLVM_COMPILER <= 20240200 && __SYCL_DEVICE_ONLY__ && __INTEL_LLVM_COMPILER < 20250100)
+#define _PSTL_TEST_COMPLEX_EXP_BROKEN                                    (__INTEL_LLVM_COMPILER <= 20240200 && __SYCL_DEVICE_ONLY__ && __INTEL_LLVM_COMPILER < 20250100)
+#define _PSTL_TEST_COMPLEX_TANH_BROKEN_IN_KERNEL                         (__INTEL_LLVM_COMPILER <= 20240200 && __SYCL_DEVICE_ONLY__ && __INTEL_LLVM_COMPILER < 20250100)

```

Comment:
It is not necessary to raise the version to `2025100` due to `__INTEL_LLVM_COMPILER <= 20240200`, is not it?

Reply:
```suggestion
#define _PSTL_TEST_COMPLEX_ACOS_BROKEN_IN_KERNEL                         (__INTEL_LLVM_COMPILER < 20250100 && __SYCL_DEVICE_ONLY__)
#define _PSTL_TEST_COMPLEX_EXP_BROKEN                                    (__INTEL_LLVM_COMPILER < 20250100 && __SYCL_DEVICE_ONLY__)
#define _PSTL_TEST_COMPLEX_TANH_BROKEN_IN_KERNEL                         (__INTEL_LLVM_COMPILER < 20250100 && __SYCL_DEVICE_ONLY__)

```